### DISCUSSION TITLE
NUnit3Aliases method missing

### DISCUSSION THREAD

**Discussion by rhtnr** [cake-build/cake#3166] [Q&A]
### What You Are Seeing?

We can specify a configuration file in NUnit3Settings. The configuration file may contain the assemblies with the tests to run. But we don't have an alias method to specify just the settings. 

All alias methods need both the settings and a list of assemblies/pattern.   

```csharp
NUnit3(testDlls,      //testDlls has the test assemblies
      new NUnit3Settings {
        Process = NUnit3ProcessOption.Multiple,
        ConfigFile = "somefile.nunit"    //somefile.nunit again has all the assemblies in var testDlls 
      //since Config file has the assemblies in it, ideally we dont have to pass param  testDlls
      });
```

### What is Expected?
When we already specify the list of assemblies containing the tests, we don't have to specify it again as a parameter to alias method. 

Therefore a new alias method is required that needs to take in only the NUnit3Settings var 

### What version of Cake are you using?
0.36

**Comment by ChrisMaddock** [comment]
I think you're misunderstanding how `ConfigFile` works here. 

That property matches up to the `--configFile` flag for the NUnit console, which is used if you want to specify an app.confg file for your test assemblies. If your using a `.nunit` file, you should just put that in the first parameter, where you currently have `testDlls`.

The only potential minor error here is that Cake calls the first parameter of the `NUnit3()` method assemblyPaths, when technically it could also include non-assembly files, such as `.nunit` files. 🙂
